README.md

```diff
@@ -98,28 +114,28 @@ Generated documentation for the latest version of the Rust crates can be accesse
 
 To build, simply run `cargo build` (or `cargo build --release`).
 
-## What's new / Changelog
+## What's New
 
 We maintain changelogs for our public-facing crates: [cedar-policy](./cedar-policy/CHANGELOG.md) and [cedar-policy-cli](./cedar-policy-cli/CHANGELOG.md).
 For a list of the current and past releases, see [crates.io](https://crates.io/crates/cedar-policy) or [Releases](https://github.com/cedar-policy/cedar/releases).
 
 ## Backward Compatibility Considerations
 
-Cedar is written in Rust and you will typically depend on Cedar via Cargo. Cargo makes sane choices for the majority of project, but your needs may differ. If you don't want automatic updates to Cedar replace
+Cedar is written in Rust and you will typically depend on Cedar via Cargo. Cargo makes sane choices for the majority of projects, but your needs may differ. If you don't want automatic updates to Cedar, then you can pin to a specific version in your `Cargo.toml`. For example:
 
-```
+```toml
 [dependencies]
-cedar-policy = "2.3.3"
+cedar-policy = "=2.4.2"
 ```
 
-with
+Note that this is different from:
 
-```
+```toml
 [dependencies]
-cedar-policy = "=2.3.3"
+cedar-policy = "2.4.2"
 ```
 
-in your `Cargo.toml` file.
+Which will pull the latest patch of version 2.4.
```

Comment:
I think this is fine to say, but it's technically more complicated than that, particularly if you have other dependencies that also depend on Cedar.  Also, it will automatically upgrade you to 2.5 or 2.6, etc, as those are semver-compatible.  It just expresses 2.4.2 is the minimum version of Cedar you accept, and you implicitly accept anything newer that is semver-compatible.  See https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html.

Reply:
Thanks for clarifying! That was my mis-understanding. I've updated the text to match your comment.